lib/performance_data/statistics.rb

```diff
@@ -0,0 +1,216 @@
+require 'json'
+require 'gds_api/performance_platform/data_out'
+
+module PerformanceData
+  class Statistics
+    attr_accessor :data_out, :content, :slug, :part_urls, :document_type, :is_multipart, :performance_data
+
+    def initialize(content, slug)
+      @data_out = GdsApi::PerformancePlatform::DataOut.new("http://www.performance.service.gov.uk")
+      @content = content
+      @slug = slug
+      @part_urls = get_part_urls(content, slug)
+      @document_type = @content["document_type"] || @content["format"]
+      @is_multipart = @part_urls.any? || (document_type == 'smart-answer')
```

Comment:
I should mention that for `simple_smart_answers`, there is not mention of that structure in the `content-schemas` repo, in the `details` section:

https://github.com/alphagov/govuk-content-schemas/blob/master/formats/simple_smart_answer/publisher/details.json

The structure I'm referring to is:

"details": {
  "parts": {
      "web_url": 'example/part-1'
   }
}